Build/Common.Build.ProjectConfiguration.props

```diff
@@ -70,19 +70,8 @@
       <Platform>ARM64</Platform>
     </ProjectConfiguration>
   </ItemGroup>
-  <ItemGroup Label="ProjectConfigurations">
-    <ProjectConfiguration Include="Debug|CHPE">
```

Comment:
Isn't this used by ChakraFull as well? If so, we will want to keep CHPE, no?

Reply:
This was cleanup; those configurations where in the props files twice.

Reply:
@Penguinwizzard It depends on which version of VC Jenkins is using, but that would be a good follow up for sure.